components/enterprise/environments.tf

```diff
@@ -0,0 +1,21 @@
+module "environment" {
+  for_each = local.environments
+
+  source               = "../../modules/environments"
+  name                 = each.key
+  common_tags          = module.environment_tags[each.key].common_tags
+  location             = var.location
+  env                  = each.key
+  asp_sku_size         = var.asp_sku_size
+  asp_sku_tier         = var.asp_sku_tier
+  product              = var.product
+  pipeline_environment = var.env
```

Comment:
not sure if I would trust the pipeline_environment, what you want is the subscription environment not what's set by the pipeline which is likely useless

Reply:
This is to ensure the OPS-ENV ADO service endpoints are created in prod and not sandbox since the sandbox pipeline cannot access the SDS subscriptions and also to the pipeline trying to create them twice. I needed a variable that read the environment in the pipeline rather than the environment being bootstrapped.

Reply:
better to just create them in the sandbox environment than do this I think but we should remove this hack